tools/acl_compressor/sources/main.cpp

```diff
@@ -193,6 +193,37 @@ static bool parse_options(int argc, char** argv, Options& options)
 	return true;
 }
 
+constexpr size_t CACHE_FLUSH_BUFFER_BYTES = 8 * 1024 * 1024;
+constexpr size_t CACHE_FLUSH_PASSES = 16;
+
+static void flush_caches(Allocator& allocator)
+{
+	uint8_t* buffer = allocate_type_array<uint8_t>(allocator, CACHE_FLUSH_BUFFER_BYTES);
+
+	for (size_t pass_index = 0; pass_index < CACHE_FLUSH_PASSES; ++pass_index)
+		std::memset(buffer, std::rand(), CACHE_FLUSH_BUFFER_BYTES);
+
+	deallocate_type_array(allocator, buffer, CACHE_FLUSH_BUFFER_BYTES);
+}
+
+static LARGE_INTEGER start_time;
+
+static void start_timer()
```

Comment:
I reworked all of this to use <chrono> in develop to remove the dependency on windows headers